tensorflow/lite/micro/examples/person_detection/README.md

```diff
@@ -171,6 +171,69 @@ From the log, we can see that it took around 170 ms to capture and read the
 image data from the camera module, 180 ms to decode the JPEG and convert it to
 greyscale, and 18.6 seconds to run inference.
 
+
+## Running on ESP32
+
+ The following instructions will help you build and deploy this sample
+ to [ESP32](https://www.espressif.com/en/products/hardware/esp32/overview)
+ devices using the [ESP IDF](https://github.com/espressif/esp-idf).
+
+ The sample has been tested on ESP-IDF version 4.0 with the following devices:
+ - [ESP32-DevKitC](http://esp-idf.readthedocs.io/en/latest/get-started/get-started-devkitc.html)
+ - [ESP-EYE](https://github.com/espressif/esp-who/blob/master/docs/en/get-started/ESP-EYE_Getting_Started_Guide.md)
+
+ ESP-EYE is a board which has a built-in camera which can be used to run this example , if you want to use other esp boards you will have to connect camera externally and write your own +image_provider.cc+ and +esp_app_camera.cc+.
+ ### Install the ESP IDF
+
+ Follow the instructions of the
+ [ESP-IDF get started guide](https://docs.espressif.com/projects/esp-idf/en/latest/get-started/index.html)
+ to setup the toolchain and the ESP-IDF itself.
+
+ The next steps assume that the
+ [IDF environment variables are set](https://docs.espressif.com/projects/esp-idf/en/latest/get-started/index.html#step-4-set-up-the-environment-variables) :
+
+  * The `IDF_PATH` environment variable is set
+  * `idf.py` and Xtensa-esp32 tools (e.g. `xtensa-esp32-elf-gcc`) are in `$PATH`
+
```

Comment:
It should also be mentioned that the `esp32-camera` component is needed.

BTW, would you recomend to clone it in `$IDF_PATH/components/` or in the `components/` dir of the example ?

Reply:
Yes I will update that in the `README` , I would recommend to clone it in `components/` dir of the example  (as we advice not to make any changes in `esp-idf` and keep it intact)